lib/emailer.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const path = require('path');
+const nodemailer = require('nodemailer');
+const stubTransport = require('nodemailer-stub-transport');
+const smtpTransport = require('nodemailer-smtp-transport');
+
+module.exports = class EmailService {
```

Comment:
I think the file/Class names are mixed up. This Class is `EmailService`, but the filename is `emailer`